src/conn/conn_dhandle.c

```diff
@@ -818,6 +818,10 @@ __wt_conn_dhandle_discard(WT_SESSION_IMPL *session)
         goto restart;
     }
 
+    /* Return from here if this was called from recovery flow. */
```

Comment:
Can you add a comment to explain the reason behind to return from here when it is called in recovery?

Reply:
Updated comments